docs/deployment/configuration.md

```diff
@@ -0,0 +1,93 @@
+# Configuration
+
+This section explain the current configuration options that are available for the different components that are deployed but the Helm charts.
+
+## Global
+
+The is only one mandatory configuration, which is the host name, for example:
+
+```yaml
+host: your-rdm-instance.com
+```
+
+Moreover, the services can be deployed along, note that it is recommended to deploy separatelly Elasticsearch and PostgreSQL for a production deployment.
+Therefore, by default only `redis` and `rabbitmq` are enabled. Example configuration:
+
+``` yaml
+postgresql:
+  inside_cluster: false
+
+elasticsearch:
+  inside_cluster: false
+```
+
+!!! info "inside_cluster availability
+    Note that the `inside_cluster` variable is supported for `redis`, `rabbitmq`, `elasticsearch`, `postgresql` and `haproxy`. The rest of the components
+    are mandatory.
+
+## HAProxy
+
+You can change the number of connections allowed by the haproxy with the `maxconn` variable:
+
+```
+haproxy:
+  maxconn: 100
+```
+
+!!! warning "Only one parent element"
+    By default the HAProxy is enabled, `inside_cluster` has the value `true`, nonetheless if you decide to set it, you only need
+    to specify once the `haproxy` parent in the yaml file. Otherwise the last one will override the previous. It should be something like:
+    ``` yaml
+    haproxy:
+        inside_cluster: true
+        maxconn: 100
+    ```
+## Nginx
+
+The charts allow you to configure the amount of connections per nginx node (replica) and the amount of nodes:
```

Comment:
1 chart <==> 1 package . "chart" is singular or is it "charts" because multiple chart are combined together?

Reply:
I have seen both in the literature, I have not seen a naming convention. Will look for it.